Examples/Scripts/PostFreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,243 @@
+##################################################################################################
+SurfaceAtlasDIR="${HCPPIPEDIR_Templates}/standard_mesh_atlases"
+GrayordinatesSpaceDIR="${HCPPIPEDIR_Templates}/91282_Greyordinates"
+GrayordinatesResolution="2" #Usually 2mm, if multiple delimit with @, must already exist in templates dir
+HighResMesh="164" #Usually 164k vertices
+LowResMeshes="32" #Usually 32k vertices, if multiple delimit with @, must already exist in templates dir
+SubcorticalGrayLabels="${HCPPIPEDIR_Config}/FreeSurferSubcorticalLabelTableLut.txt"
+FreeSurferLabels="${HCPPIPEDIR_Config}/FreeSurferAllLut.txt"
+ReferenceMyelinMaps="${HCPPIPEDIR_Templates}/standard_mesh_atlases/Conte69.MyelinMap_BC.164k_fs_LR.dscalar.nii"
+RegName="MSMSulc" #MSMSulc is recommended, if binary is not available use FS (FreeSurfer)
+
+# Requirements for this script
+#  installed versions of: FSL, Connectome Workbench (wb_command)
+#  environment: HCPPIPEDIR, FSLDIR, CARET7DIR
+
+# Log the originating call
+echo "$@"
+
+#NOTE: syntax for QUEUE has changed compared to earlier pipeline releases,
+#DO NOT include "-q " at the beginning
+#default to no queue, implying run local
+QUEUE=""
+#QUEUE="hcp_priority.q"
+
+########################################## INPUTS ########################################## 
+#Scripts called by this script do assume they run on the outputs of the longitudinal FreeSurfer Pipeline
+######################################### DO WORK ##########################################
+
+Template_list=( ${Template_list[@]} )
+Timepoint_list=( ${Timepoint_list[@]} )
+Subjlist=( ${Subjlist[@]} )
+
+#iterate over all subjects.
+for i in ${!Subjlist[@]}; do
+  Subject=${Subjlist[i]}
+  LongitudinalTemplate=${Template_list[i]}
+  IFS=@ read -r -a Timepoints <<< "${Timepoint_list[i]}"
+  
+  echo Subject: $Subject
+  echo Template: $LongitudinalTemplate
+  echo Timepoints: ${Timepoints[@]}
+
+  if [[ "${command_line_specified_run_local}" == "TRUE" || "$QUEUE" == "" ]] ; then
+      echo "About to locally run longitudinal mode of ${HCPPIPEDIR}/PostFreeSurfer/PostFreeSurferPipeline.sh "
+      #NOTE: fsl_sub without -q runs locally and captures output in files
+      queuing_command=("$FSLDIR/bin/fsl_sub")
+  else
+      echo "About to use fsl_sub to queue longitudinal mode of ${HCPPIPEDIR}/PostFreeSurfer/PostFreeSurferPipeline.sh"
+      queuing_command=("$FSLDIR/bin/fsl_sub" -q "$QUEUE")	
+  fi
+
+  ##########################################################################################
+  # PostFreesurferPipelineLongPrep.sh processing
+  ##########################################################################################
+  #process timepoints
+  job_list=()
+  for TP in ${Timepoints[@]}; do
+	echo "Running ppFS-long for timepoint: $TP"
+        job=$($queuing_command ${HCPPIPEDIR}/PostFreeSurfer/PostFreesurferPipelineLongPrep.sh --subject="$Subject" --path="$StudyFolder" \
```

Comment:
Prep is being called both in here and in its own Batch?

Reply:
Prep's own batch is an early version, before prep batch was merged into PostFreeSurferPipelineBatch-long.sh. 
I'd like to keep it for dev/testing purposes.